src/depth-to-rgb-calibration.h

```diff
@@ -43,7 +44,8 @@ namespace librealsense
         stream_profile_interface * get_to_profile() const { return _to; }
         rs2_dsm_params const & get_dsm_params() const { return _dsm_params; }
 
-        rs2_calibration_status optimize( std::function<void( rs2_calibration_status )> call_back = nullptr );
+        rs2_calibration_status optimize( std::function<void( rs2_calibration_status )> call_back = nullptr ,
+            std::function<bool()> should_continue = nullptr);
```

Comment:
Shouldn't be needed (and you don't use it) -- leftover?

Reply:
Are you regarding 
std::function<void( rs2_calibration_status )> call_back = nullptr ?
It being used in the unit tests so I did not touched it.

leftovers?